tachyon/c/zk/air/sp1/baby_bear_poseidon2_opened_values.cc

```diff
@@ -12,8 +13,8 @@ using OpenedValues =
     std::vector<std::vector<std::vector<std::vector<math::BabyBear4>>>>;
 
 tachyon_sp1_baby_bear_poseidon2_opened_values*
-tachyon_sp1_baby_bear_poseidon2_opened_values_create() {
-  return c::base::c_cast(new OpenedValues());
+tachyon_sp1_baby_bear_poseidon2_opened_values_create(size_t rounds) {
+  return c::base::c_cast(new OpenedValues(rounds));
```

Comment:
I don't get why this change is in this commit.

Reply:
All `xxx_allocate()` C APIs accept a `size_t` argument for round, which indicates the index of the round. This implies that these vectors are expected to be resized according to rounds beforehand. Similarly, `tachyon_sp1_baby_bear_poseidon2_opened_values_allocate_outer()` and `tachyon_sp1_baby_bear_poseidon2_opened_values_allocate_inner()` require that the vectors be resized during construction.